cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,118 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('display multiple emails "unverified" and "verified"', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    describe('Emails edit modals' + runInfo(), () => {
```

Comment:
Should the email list changes be saved right after those are done by the user (as it currently works) or only after the user clicks the save button (as most other modals currently work)?.